cmake/GoogleTest.cmake.in

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 3.4.0)
```

Comment:
this explicitly bumps us up. was this deliberate?

i think this means that if you want to enable testing (which we do by default) you need 3.4.0. Otherwise you can use 2.8. That's weird but acceptable if it enables the global version bump as a follow-up.

Reply:
Not really intentional. Should still work with 2.8, let's unbump.